api/envoy/config/core/v3/base.proto

```diff
@@ -303,12 +303,31 @@ message RuntimeFeatureFlag {
   string runtime_key = 2 [(validate.rules).string = {min_len: 1}];
 }
 
+// Please use :ref:`KeyValuePair <envoy_api_msg_config.core.v3.KeyValuePair>` instead.
+// [#not-implemented-hide:]
```

Comment:
Will this still be ok with the docs links generation (the KeyValue will be hidden from the docs, but there will be many fields of that type pointing to nowhere).
I think a deprecated message annotation can be added instead.

Reply:
Actually, the `KeyValue`, `KeyValueMutation`, etc was never be used at any other where. So, I think hide it directly may be more clean.